frontend/src/pages/Teachers.vue

```diff
@@ -4,44 +4,6 @@
       <a class="logo">Teachers</a>
     </PageHeader>
 
-    <section>
-      <b-table
-        :data="teachersData"
-        :selected.sync="selected"
-        @dblclick="goToTeacher"
```

Comment:
This deletes the table with the `dblclick` event handler. Let's merge this PR and reintroduce the event handler on the surviving table. (Refer to #126 which was closed in favour of this PR.)

Reply:
Can we take this as an approval?

Reply:
We edited this PR to reintroduce the event handler.
Also took the chance to update the style for the table (all text in same row will turn white when selected, except for status description) for readability.